google-cloud-spanner/src/test/java/com/google/cloud/spanner/GceTestEnvConfig.java

```diff
@@ -51,6 +61,8 @@ public GceTestEnvConfig() {
     double errorProbability =
         Double.parseDouble(System.getProperty(GCE_STREAM_BROKEN_PROBABILITY, "0.0"));
     checkState(errorProbability <= 1.0);
+    boolean attemptDirectPath = Boolean.getBoolean(ATTEMPT_DIRECT_PATH);
```

Comment:
Should this be
```suggestion
    boolean attemptDirectPath = Boolean.getBoolean(System.getProperty(ATTEMPT_DIRECT_PATH));
```

Reply:
Good point, actually that's what I initially did, but after I checked the [doc](https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html#getBoolean-java.lang.String-), it seems this method would call getProperty for us:
> Returns true if and only if the system property named by the argument exists and is equal to the string "true". (Beginning with version 1.0.2 of the JavaTM platform, the test of this string is case insensitive.) A system property is accessible through getProperty, a method defined by the System class.
If there is no property with the specified name, or if the specified name is empty or null, then false is returned.